src/core/FilePath.cpp

```diff
@@ -211,6 +211,9 @@ FilePath::FilePath()
     else if (testSetDir(appDirPath + "/share")) {
     }
 #endif
+    // Last ditch test when running in the build directory (mainly for travis tests)
+    else if (testSetDir(QString(KEEPASSX_SOURCE_DIR) + "/share")) {
```

Comment:
No, it doesn't work that way. KEEPASSX_PREFIX_DIR isn't correctly set and I prefer to keep my stable KeePassXC and my development KeePassXC separate. Sure, I could make install my dev version to some other place, but then I also have to remember to do it every time something changes in the data dir. Things like that can cost you hours of your time when you can't find out, why something is not working the way it is supposed to. ;-)

Removing th QT_DEBUG stuff from the beginning may work, but if you have a different version of KeePassXC installed in KEEPASSX_PREFIX_DIR this might cause conflicts. I would prefer to always check KEEPASSX_SOURCE_DIR first for Debug builds.